packages/forma-36-react-components/src/components/CheckboxField/README.mdx

```diff
@@ -0,0 +1,57 @@
+# CheckboxField
+
+CheckboxField is a component that gives an possiblity to easy setup proper checkbox with its label. Component also contains an options that allows setting helper text displayed below the label. You might see checkboxes in forms, they might be also used for filtering data on the page or terms and conditions. Everywhere where user have multiple choice you might want to use this component.
+
+## Example of usage
+
+```jsx
+import {CheckboxField} from '@contentful/forma-36-react-components';
+
+<CheckboxField
+  checked={}
+  onChange={hadnleOnChange}
+  name="checkbox name"
+  id="randomId"
+  labelText="Label text"
+  helpText="Helper text displayed below"
+/>
+```
```

Comment:
Shouldn't we start every doc with the simplest way of using the component? Is this the simplest way? I mean, do I need to pass all these props to see a checkbox on my screen?

Reply:
yeah, make sense. On my way. Thx!

Reply:
Fixed, thx for feedback 🙇‍♀️